net.line72.campcounselor.yml

```diff
@@ -0,0 +1,47 @@
+app-id: net.line72.campcounselor
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: campcounselor
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /share/aclocal
+  - /man
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+  - /lib/girepository-1.0
+  - /share/doc
+  - /share/gir-1.0
+modules:
+  - name: intltool
```

Comment:
please use the shared-modules.

https://github.com/flathub/shared-modules/

Reply:
done.